auth-web/src/components/auth/account-settings/product/ProductPackage.vue

```diff
@@ -115,19 +137,49 @@
 </template>
 
 <script lang="ts">
-import { AccessType, Account, ProductStatus, Role } from '@/util/constants'
-import { AccountFee, OrgProduct, OrgProductCode, OrgProductFeeCode, OrgProductsRequestBody, Organization } from '@/models/Organization'
+import {
+  AccessType,
+  Account,
+  AccountStatus,
+  Product as ProductEnum,
+  ProductStatus,
+  Role,
+  TaskType
+} from '@/util/constants'
+import {
+  AccountFee,
+  OrgProduct,
+  OrgProductCode,
+  OrgProductFeeCode,
+  OrgProductsRequestBody,
+  Organization
+} from '@/models/Organization'
 import { Action, State } from 'pinia-class'
 import { Component, Mixins } from 'vue-property-decorator'
 import AccountChangeMixin from '@/components/auth/mixins/AccountChangeMixin.vue'
+import CautionBox from '@/components/auth/common/CautionBox.vue'
 import { KCUserProfile } from 'sbc-common-components/src/models/KCUserProfile'
 import ModalDialog from '@/components/auth/common/ModalDialog.vue'
 import Product from '@/components/auth/common/Product.vue'
+import { ProductStatusMsgContentIF } from '@/models/external'
 import { useOrgStore } from '@/stores/org'
 import { useUserStore } from '@/stores/user'
+import { userAccessDisplayNames } from '@/resources/QualifiedSupplierAccessResource'
 
 @Component({
```

Comment:
if you have the spare cycles, comp-api, if not no worries

Reply:
I can do this!